src/Watchlists/WatchListItem.jsx

```diff
@@ -34,18 +34,18 @@ export default function WatchListItem({
       <View style={styles.watchListItemColumn}>
         {dailyChange > 0 ? (
           <Text style={[styles.watchListItemColumnText, { color: '#21af78' }]}>
-            {`+${dailyChange}%`}
+            {`+${round(dailyChange)}%`}
```

Comment:
I would round in the reducer or action, so we will display the same number everywhere and the application will be more consistent. Does it have any benefit if we round the number here?